### ISSUE TITLE
meta-commands

### ISSUE THREAD

**Issue by ndurks** [facebook/mcrouter#335]
[memcached](https://github.com/memcached/memcached) recently implemented a series of [meta-commands](https://github.com/memcached/memcached/wiki/MetaCommands), I am interested in using these and would like to see support in mcrouter. Is there any preexisting work or interest in making this happen? What is the likelihood of a meta-command PR getting accepted?

**Comment by shivnagarajan** [comment]
@dormando also, as he did most of the work on the memcached side for the Meta Commands.

**Comment by jjhuo** [comment]
yes, some of those meta-commands are very useful, that'll be great if mcrouter can support them.
E.g. https://github.com/memcached/memcached/wiki/MetaCommands#atomic-stampeding-herd-handling
